test/tests/kernels/vector_dot_dot/vector_test.i

```diff
@@ -0,0 +1,85 @@
+# Tests calculation of first and second time derivative
+# of a coupled vector variable in a material
+# a_vec(x,y,z,t) = [t*(t*x + y), t*y, 0]
+# a_vec_dot(x,y,z,t) = [2*t*x + y, y, 0]
+# a_vec_dot_dot(x,y,z,t) = [2*x, 0, 0]
+#
+# IMPORTANT NOTE:
+# Currently, this test produces a_vec_dot and a_vec_dot_dot that contains oscillations over time.
+# This is a known by-product of Newmark Beta time integration (see the Newmark Beta documentation),
+# but as of Summer 2019, there is no alternative time integrator in MOOSE that can contain these
```

Comment:
Did you mean dampen instead of contain at the end of this line?

Reply:
Yes! Thanks for the catch.